docs/admin/deploy/docker-compose/upgrade.mdx

```diff
@@ -59,70 +59,84 @@ $ docker-compose up -d --remove-orphans
+> <Callout type="warning">Multiversion upgrades **require downtime**, please see our [cautionary note](/admin/updates/#best-practices) on upgrades, if you have any concerns about running a multiversion upgrade, please reach out to us at [support@sourcegraph.com](mailto:support@sourcegraph.com) for advisement. </Callout>
 
----
-
-> **⚠️ Attention:** please see our [cautionary note](/admin/updates/#best-practices) on upgrades, if you have any concerns about running a multiversion upgrade, please reach out to us at [support@sourcegraph.com](mailto:support@sourcegraph.com) for advisement.
-
-To perform a **manual** multi-version upgrade on a Sourcegraph instance running on Docker compose follow the procedure below:
+To perform a multi-version upgrade via migrators [upgrade](/admin/updates/migrator/migrator-operations#upgrade) command on a Sourcegraph instance running on Docker compose follow the procedure below:
 
 1. **Check Upgrade Readiness**:
    - Check the [upgrade notes](/admin/updates/docker_compose#docker-compose-upgrade-notes) for the version range you're passing through.
    - Check the `Site Admin > Updates` page to determine [upgrade readiness](/admin/updates/#upgrade-readiness).
 
-2. **Disable Connections to the Database**:
-   - Run the following command in the directory containing your `docker-compose.yaml` file.
-  ```sh
-  $ docker-compose stop && docker-compose up -d pgsql codeintel-db codeinsights-db
-  ```
-3. **Run Migrator with the `upgrade` command**:
-   - The following procedure describes running migrator in brief, for more detailed instructions and available command flags see our [migrator docs](/admin/updates/migrator/migrator-operations#docker-compose).
-    1. Set the migrator `image:` in your `docker-compose.yaml` to the **latest** release of `migrator`. **Example:**
+2. **Pull and merge upstream changes**:
+   - Follow the [standard upgrade procedure](#standard-upgrades) to pull and merge upstream changes from the version you are upgrading to to your `release` branch.
+
+3. **Disable Connections to the Database**:
+   - Stop running containers connected to the databases and the databases:
+   ```
+   $ docker-compose down --remove-orphans
+   ```
+
+4. **Run Migrator with the `upgrade` command**:
+   - *For more detailed instructions and available command flags see our [migrator docs](/admin/updates/migrator/migrator-operations#upgrade).*
+    1. If the migrator `image:` in your `docker-compose.yaml` wasn't updated to in the **latest** release of `migrator` in step 2 set migrator's image to the latest release.
+    **Example:**
     ```yaml
     migrator:
       container_name: migrator
-      image: 'index.docker.io/sourcegraph/migrator:5.0.4'
+      image: 'index.docker.io/sourcegraph/migrator:6.0.0'
```

Comment:
we may want to use `CURRENT_VERSION_STRING` stuff here?

Reply:
I'd be into doing this as a revision later yea, at the moment I think we just need to ship this, and a lot of SG uses this stuff